src/chia_dialect.rs

```diff
@@ -2,6 +2,7 @@ use crate::allocator::{Allocator, NodePtr};
 use crate::core_ops::{op_cons, op_eq, op_first, op_if, op_listp, op_raise, op_rest};
 use crate::cost::Cost;
 use crate::dialect::Dialect;
+use crate::dialect::{NO_NEG_DIV, NO_UNKNOWN_OPS};
```

Comment:
These flags belong in this file, not in `dialect.rs`, as they only modify `ChiaDialect`.